main.py

```diff
@@ -203,6 +206,48 @@ def DisplayTilesInHand(PlayerTiles):
     print("Your current hand:", PlayerTiles)
 
 
+def DisplayLeaderboard():
+    ScoreList = LoadScores()
+    RowNum = 0
+    print()
+    print("--------------")
+    print("TOP 10 PLAYERS")
+    print("--------------")
+    print()
+    print("Pos Name Score Date")
+    for Name, Score, Date in ScoreList:
+        if RowNum == 0:
+            RowNum += 1
+        else:
+            print(str(RowNum) + ".", Name, Score, Date)
+            RowNum += 1
+    print()
+    input("Press Enter to return to the main menu")
+
+
+def LoadScores():
+    with open("scores.csv") as ScoresCSV:
+        Scores = csv.reader(ScoresCSV)
+        ScoreList = sorted(Scores, key=operator.itemgetter(1), reverse=True)
+        return ScoreList[:11]
+
+
+def GetPlayerName(PlayerNumber):
+    PlayerName = ""
+    print()
+    while not 0 < len(PlayerName) <= 20:
```

Comment:
Don't see a reason why we should limit player name length, but if you insist do `while len>20 or len==0`

Reply:
I was going to use it to help format the table, but it helps prevent ridiculously long names ruining the table when printed